content/source/docs/cloud/api/ip-ranges.html.md

```diff
@@ -38,17 +40,22 @@ Name                             | Type   | Description
 
 -> **Note:** The IP Ranges API does not require authentication
 
+-> **Note:** This endpoint supports [If-Modified-Since][] HTTP request header
+
 `GET /meta/ip-ranges`
 
 Status  | Response                                        | Reason
 --------|-------------------------------------------------|----------
 [200][] | `application/json`                              | The request was successful
+[304][] | empty body                                      | The request was successful, IP ranges were not modified since the specified date in `If-Modified-Since` header
```

Comment:
It doesn't *really* matter since this just gets parsed into Markdown, but I can't help but notice how unnecessarily wiiiiiiiiiide this table is :) Not your fault, but it's more obvious with a long description in one of the cells.